features/generator_features.py

```diff
@@ -171,15 +171,28 @@ class IncrementalRotation(traits.HasTraits):
     delta_rotation_x = traits.Float(0.0, desc="rotation step size about bmi3d x-axis in degrees")
 
     trials_per_increment = traits.Int(1, desc="number of successful trials per rotation step")
+    final_tracking_out_time = traits.Float(1.6, desc="Time allowed to be tracking outside the target for final rotation") # AKA tolerance time
```

Comment:
so tracking_out_time is applied to all rotations except the final one? and then it switches? just making sure i understand correctly

Reply:
Yes